Doc/tutorial/modules.rst

```diff
@@ -48,7 +48,8 @@ command::
    >>> import fibo
 
 This does not enter the names of the functions defined in ``fibo``  directly in
```

Comment:
```suggestion
This does not add the names of the functions defined in ``fibo``  directly to
```

The old verbiage doesn't really make sense after replacing "symbol table" with "namespace"